src/main.rs

```diff
@@ -14,13 +15,33 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
-    let warnings = dotenv_linter::run(args, &current_dir)?;
+    let warnings = dotenv_linter::run(&args, &current_dir)?;
     if warnings.is_empty() {
         process::exit(0);
     }
 
+    let no_color = args.is_present("no-color");
+    let should_print_total_problems = !no_color;
+    if no_color {
+        control::set_override(false);
+    }
+
     warnings.iter().for_each(|w| println!("{}", w));
-    process::exit(1);
+    if should_print_total_problems {
+        print_total_problems(warnings.len())
+    }
+    process::exit(1)
+}
+
+fn print_total_problems(total_problems: usize) {
+    println!();
+    println!(
+        "{}",
+        format!("{} {} problems", "\u{2717}", total_problems)
+            .red()
+            .bold()
+    );
```

Comment:
@DDtKey sorry, this refactor dismissed your review :grimacing:

Reply:
> @DDtKey sorry, this refactor dismissed your review grimacing

Good catch! 
Thanks, it's okay :relaxed: